### PR TITLE
Remove Nimue and update to build tools 17.2.5

### PR DESCRIPTION
**Pull request by jeanplevesque**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Build or CI related changes

## Description

- Remove Nimue and update to build tools 17.2.5

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

**Review by vincentcastagna — APPROVED** [review]

File: build/steps-build.yml [hunk 1]
```diff
@@ -147,6 +170,27 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: "${{ parameters.pathToSrc }}/coverage.cobertura.xml"
 
+- task: CopyFiles@2
+  displayName: 'Copy AAB'
+  inputs:
+    Contents: '**/*.aab'
```
Comment (vincentcastagna) [inline build/steps-build.yml:176]: I think you want to filter with the `-Signed`

Comment (jeanplevesque) [reply build/steps-build.yml:176]: Do you have an example of that?

Comment (vincentcastagna) [reply build/steps-build.yml:176]: Not here, in appcenter task my bad :p

**Pull request merged**